SonarQube.MSBuild.Tasks/Docs/IntegrationTargetDependencies.dgml

```diff
@@ -0,0 +1,96 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DirectedGraph xmlns="http://schemas.microsoft.com/vs/2009/dgml">
```

Comment:
Documentation. DGML file that shows the dependencies between the various targets.